gradle-module/src/main/java/org/example/Main.java

```diff
@@ -22,4 +23,11 @@ public static void main(String[] args) {
     private static void test() {
         System.out.println("test");
     }
-}
\ No newline at end of file
+
+   /* asd
+    we
+    asd
+
+    */
+    
+}
```

Comment:
Ensure that there is a newline at the end of the file. This is a common convention in many programming languages and can help prevent issues with certain tools and editors.